src/allmydata/client.py

```diff
@@ -220,6 +257,9 @@ def create_client_from_config(config, _client_factory=None):
 
     :param _client_factory: for testing; the class to instantiate
         instead of _Client
+
+    :param introducer_factory: for testing; the class to instantiate instead
+        of IntroducerClient
     """
```

Comment:
`_client_factory` starts with an underscore I think to *ahem* underscore the fact that it's internal .. maybe `introducer_factory=` should too?